scripts/generate_package_diffs_for_flavors/generate_package_diffs_for_flavors.py

```diff
@@ -20,10 +20,17 @@ def parse_package_list_file(file_path: Path) -> str:
     return result_string
 
 
+def check_for_duplicated_packages(df: pd.DataFrame):
+    duplicates = df.duplicated(subset=["Package"])
+    if any(duplicates):
+        raise ValueError(f"Found duplicated packages, see package list {df}")
```

Comment:
now I would like to have a Github action which runs this, because so we get early feedback